From 1acc2fd48c755a8931fa87b8d0560b750316059f Mon Sep 17 00:00:00 2001
From: "Jose M. Guisado Gomez" <guigom@riseup.net>
Date: Wed, 22 Feb 2023 11:20:55 +0100
Subject: py: replace distutils with setuptools

Removes a deprecation warning when using distutils and python >=3.10.

Python distutils module is formally marked as deprecated since python
3.10 and will be removed from the standard library from Python 3.12.
(https://peps.python.org/pep-0632/)

From https://setuptools.pypa.io/en/latest/setuptools.html

"""
Packages built and distributed using setuptools look to the user like
ordinary Python packages based on the distutils.
"""

Signed-off-by: Jose M. Guisado Gomez <guigom@riseup.net>
Signed-off-by: Florian Westphal <fw@strlen.de>
---
 py/setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

(limited to 'py')

diff --git a/py/setup.py b/py/setup.py
index 72fc8fd9..8ad73e7b 100755
--- a/py/setup.py
+++ b/py/setup.py
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-from distutils.core import setup
+from setuptools import setup
 from nftables import NFTABLES_VERSION
 
 setup(name='nftables',
-- 
cgit v1.2.3
